langgraph/src/pregel/index.ts

```diff
@@ -198,51 +206,83 @@ export class Pregel
   // Because Pregel extends `Runnable`.
   lc_namespace = ["langgraph", "pregel"];
 
+  nodes: Record<string, PregelNode>;
+
   channels: Record<string, BaseChannel> = {};
 
-  output: string | Array<string> = "output";
```

Comment:
`Pregel` interface updates:
1. `output` --> `outputChannels`
2. `input` --> `inputChannels`
3. `hidden` is removed
4. `interrupt` --> `interruptAfterNodes` and `interruptBeforeNodes`
5. `streamMode` is added (unused right now)
6. `streamChannels` is added
7. `autoValidate` is added (unused right now)